pkgs/by-name/cn/cnsprcy/package.nix

```
@@ -5,19 +5,29 @@
   sqlite,
 }:
 
-rustPlatform.buildRustPackage {
+rustPlatform.buildRustPackage (finalAttrs: {
   pname = "cnsprcy";
-  version = "0.2.0";
+  version = "0.3.0";
 
   src = fetchFromSourcehut {
     owner = "~xaos";
     repo = "cnsprcy";
-    rev = "v0.2.0";
-    hash = "sha256-f+DauSU4bT3EljY8/ig7jLnUgyDPEo2NSBQcPN0iKx0=";
+    tag = "v${finalAttrs.version}";
+    hash = "sha256-OBNl2ZZi1jmJjJvzCwaIN9DSYjGaEo0g4o7/vVOF5aQ=";
```

This will point to the branch which has the binary crate (thanks @supinie for figuring this one out):

```suggestion
    rev = "cnspr/v${finalAttrs.version}";
    hash = "sha256-XK7eNO8J/2E5qDVJLDKHB/8kn/qSyVIr1OW63PNBT5k=";
```